build/azure-pipelines.yml

```diff
@@ -100,7 +108,7 @@ stages:
 
 - stage: Build_Production
   dependsOn: Build_Staging
-  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
+  condition: and(succeeded(), eq(variables['IsPullRequestBuild'], 'false'))
```

Comment:
dont we want to use the internalfirebaseOptions when we make non-prod internal builds?

Reply:
The comment is probably misaligned, but I'm sure I'm using internal for staging.

Reply:
![image](https://github.com/nventive/FlutterApplicationTemplate/assets/57812285/41927e1a-137e-4296-910e-9c0f9cba2974)

Reply:
Oh you're right